javascript-arrays/main.js

```diff
@@ -0,0 +1,22 @@
+var colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+console.log('America is', colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.');
+
+colors[2] = 'green';
+console.log('Mexico is', colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.');
+console.log('value of colors:', colors);
+
+var students = ['John', 'Bob', 'Jon', 'Bobby'];
+var numberOfStudents = students.length;
+
+console.log('There are', numberOfStudents, 'students in the class.');
+
+var lastIndex = numberOfStudents - 1;
+lastIndex = students[3];
+var lastStudent = lastIndex;
```

Comment:
here, you want to use the length property -1 to find the end of the array, and then use that variable to access the final index of the array and save that value into a variable. give it a shot!